src/prefect/deployments/base.py

```diff
@@ -416,6 +416,9 @@ def _format_deployment_for_saving_to_prefect_file(
         elif isinstance(deployment["schedule"], BaseModel):
             deployment["schedule"] = deployment["schedule"].dict()
 
+        if "is_schedule_active" in deployment:
+            deployment["schedule"]["active"] = deployment.pop("is_schedule_active")
+
```

Comment:
can you remind me why we wouldn't want to store it in the `prefect.yaml`? it seems like it'd be nice to have for when you re-deploy with `deploy --all`, otherwise i think that the schedules would all be active again.